cs-minibot-platform-src/src/main/resources/public/scenario/scenario2.txt

```diff
@@ -0,0 +1 @@
+[{"type":	"simulator.simbot",		"position":"[3,3]", "angle":0},{"type":	"scenario_object",		"position":"[5,5]", "angle":45, "size":1},{"type": "scenario_object", "position":"[3,3]", "angle":45, "size": 1}]
```

Comment:
Can we format this better. Like
[
{
"type": "simulator.simbot",
"position": "[3,3]", "angle": 0
},
 {
"type": "scenario_object",
etc....

]